package/MDAnalysis/coordinates/base.py

```diff
@@ -137,6 +137,14 @@
 from . import core
 from .. import NoDataError
 
+from ..auxiliary.base import AuxReader
+from ..auxiliary.xvg import XVGReader  # ?
+
+class Namespace(object):
+    # set up a basic class so we can make an 'aux' namespace in Timestep 
+    # for auxiliary data; might be better defined elsewhere...?
+    # There's probably a better way to do this
```

Comment:
Seems to me as the good way and the good place to do this, as there seems to be a consensus around the namespace vs the dictionary.  The class can be moved latter to `.lib.util` if it get duplicated.